src/main/scala/GamePrompts.scala

```diff
@@ -0,0 +1,49 @@
+import GameHelpers.{calculateWordScore}
+
+object GamePrompts {
+
+  def promptName(): String = {
+    println("\u001b[33;1mWhat is your name?")
```

Comment:
No closing tag here means that the colour code persists when the user provides their input. Looks pretty, but not sure if you intended that or not!